doc/src/devdocs/libgit2.md

```diff
@@ -0,0 +1,48 @@
+# Base.LibGit2
+
+The LibGit2 module provides bindings to [libgit2](https://libgit2.github.com/), a portable C library that
+implements core functionality for the Git version control system.
+These bindings are currently used to power Julia's package manager.
+It is expected that this module will eventually be moved into a separate package.
+
+### Functionality
+
+Some of this documentation assumes knowledge of the libgit2 API.
+For more information on some of the referenced objects, consult the upstream
```

Comment:
In this case I could also just omit the word "objects" and simply restructure the sentence to "For more information, consult the upstream libgit2 API reference."